### PR TITLE
Burgerspace: Added recipe and icon for 1.9.2

### PR DESCRIPTION
**Pull request by lich**
Before merging this recipe, I suggest to merge update for it's main dependency:
https://github.com/haikuports/haikuports/pull/381
Google Code-in Task:
https://codein.withgoogle.com/dashboard/task-instances/5220531850706944/?sp-page=1

### CODE REVIEW

File: games-arcade/burgerspace/patches/burgerspace-1.9.2.patchset [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+index 78e3d69..c888df2 100755
+--- a/configure
++++ b/configure
+@@ -3020,7 +3020,8 @@ AMTAR=${AMTAR-"${am_missing_run}tar"}
+ 
+ am__tar='${AMTAR} chof - "$$tardir"'; am__untar='${AMTAR} xf -'
+ 
+-
++# Haiku needs lnetwork ld flag
++LDFLAGS="-lnetwork"
```
Comment (diversys) [inline games-arcade/burgerspace/patches/burgerspace-1.9.2.patchset]: It's probably easier to add LDFLAGS before runConfigure

File: games-arcade/burgerspace/burgerspace-1.9.2.recipe [hunk 2]
```diff
@@ -0,0 +1,58 @@
+SUMMARY="A hamburger-smashing video game"
+DESCRIPTION="A game in which you are a chef, and must walk over hamburger in \
```
Comment (pulkomandy) [inline games-arcade/burgerspace/burgerspace-1.9.2.recipe]: in \n gredients? It looks like you cut the description in the middle of a word here?

File: games-arcade/burgerspace/burgerspace-1.9.2.recipe [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+BUILD()
+{
+	runConfigure ./configure LDFLAGS='-lnetwork'
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	addResourcesToBinaries $portDir/additional-files/burgerspace.rdef $binDir/burgerspace
```
Comment (diversys) [inline games-arcade/burgerspace/burgerspace-1.9.2.recipe]: It should be probably moved to $appsDir/BurgerSpace/BurgerSpace

File: games-arcade/burgerspace/additional-files/burgerspace.rdef [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+resource app_version {
+       major  = 0,
+       middle = 7,
+       minor  = 4,
+
+       variety = B_APPV_FINAL,
+       internal = 0,
+
+       short_info = "Burgerspace",
+       long_info = "An game about a chef fighting against food."
```
Comment (humdingerb) [inline games-arcade/burgerspace/additional-files/burgerspace.rdef]: "A game..."

File: games-arcade/burgerspace/additional-files/burgerspace.rdef [hunk 5]
```diff
@@ -0,0 +1,457 @@
...
+
+resource app_version {
+       major  = 0,
+       middle = 7,
+       minor  = 4,
+
+       variety = B_APPV_FINAL,
+       internal = 0,
+
+       short_info = "Burgerspace",
```
Comment (diversys) [inline games-arcade/burgerspace/additional-files/burgerspace.rdef]: BurgerSpace

File: games-arcade/burgerspace/burgerspace-1.9.2.recipe [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+SOURCE_URI="http://perso.b2b2c.ca/sarrazip/dev/burgerspace-$portVersion.tar.gz"
+CHECKSUM_SHA256="34b0feccf543b11f9f9a182203d67cd453675255fed4b745e4e74c121b211bf2"
+SOURCE_DIR="burgerspace-$portVersion"
+ADDITIONAL_FILES="burgerspace.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	burgerspace = $portVersion
```
Comment (pulkomandy) [inline games-arcade/burgerspace/burgerspace-1.9.2.recipe]: You need a $secondaryArchSuffix here

**Pull request merged**